Makefile

```diff
@@ -37,6 +37,10 @@ gen-testdata:
 	@$(GOTEST) -v ./pkg/defkinds/webserver -testdata
 	@echo "WARNING: Be sure to review generated testdata files before committing them."
 
+.PHONY: gen-diagrams
+gen-diagrams: build install
```

Comment:
:thinking: are you sure you want to execute the build and install tasks each time you want to generate the diagrams ?

What about letting this taks pre-tasks free and only call the pre-tasks when needed ? e.g.
```
$ make build install gen-diagrams
```

Reply:
Mh yeah, these tasks take no time when there's no change in the source code and it forces the recompilation of the binary if there was changes. As such, diagrams are always generated with the latest version of the source code :wink: